src/components/GroupElements/GroupElements.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import classes from './GroupElements.module.css';
+
+export interface GroupElementsProps {
+  /** The list of elements to place besides eachother*/
+  children: React.ReactNode;
+}
+
+/**
+ * @component
+ * A component that displays elements side by side within a container. You can place as many components besides eachother as you want. 
+ * You must have at least 2 components as children for the component to work.
+ 
+ * @example
+ * <GroupElements>
+ *  <Button>Previous</Button>
+ *  <Button>Next</Button>
+ * </GroupElements>
+ */
+
+export const GroupElements = ({ children }: GroupElementsProps) => {
+  return (
+    <ul className={classes.list}>
+      {children.map((child: React.ReactNode, index: number) => (
+        <li
```

Comment:
Why display them as list items?

Reply:
Design system does that for chip group and I think it is a common way to show semantically that two components are within the same group.

Reply:
Ok, that makes sense 👍